### ISSUE TITLE
Menu - when user on page that is in the menu the focus state fails colour contrast

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#188]
see screenshot

![Menu_-_when_user_on_page_that_is_in_the_menu_the_focus_state_fails_colour_contrast](https://user-images.githubusercontent.com/2722522/65949443-21528800-e434-11e9-9666-8728987fa9bc.png)
